src/pages/itemListPage/BestItemListSection.js

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from "react";
+
+import Item from "./components/Item.js";
+
+import { useMediaQuery } from "../../hooks/useMediaQuery.js";
+import { useApi } from "../../hooks/useApi.js";
+import { getProducts } from "../../apis/apis.js";
+import "./BestItemListSection.css";
+
+const pageSizeTable = { PC: 4, TABLET: 2, MOBILE: 1 };
+
+export default function BestItemListSection() {
+  const media = useMediaQuery();
+  const [paramObj, setParamObj] = useState({
+    pageSize: pageSizeTable[media],
+    orderBy: "favorite",
+  });
+  const [isLoading, error, data] = useApi(getProducts, paramObj);
+
+  useEffect(() => {
+    setParamObj({ pageSize: pageSizeTable[media], orderBy: "favorite" });
```

Comment:
P2:
orderBy 값이 중복되니 아래처럼 작성하시는 걸 추천드립니다.

```suggestion
    setParamObj((prev) => ({ ...prev, pageSize: pageSizeTable[media]}));
```